tests/AppBundle/EdgeToEdge/UserLoginTest.php

```diff
@@ -69,4 +73,76 @@ public function testUserLoginFrontend()
         $this->isSuccessful($client->getResponse());
         $this->assertTrue($crawler->filter('Anmelden')->count() === 0, 'Button "Anmelden" found');
     }
+
+    /**
+     * @dataProvider urlProvider
```

Comment:
Since $url is a DataProvider the typhint "@dataProvider urlProvider" is correct in the methods phpdoc block